src/core/server/http/logging/get_response_log.ts

```diff
@@ -95,6 +98,7 @@ export function getEcsResponseLog(request: Request, log: Logger) {
     user_agent: {
       original: request.headers['user-agent'],
     },
+    trace: traceId ? { id: traceId } : undefined,
```

Comment:
This made me wonder if there's a recommendation for ECS to avoid including fields altogether if they are not defined. I didn't see anything in the docs so I assume this is fine... it obviously doesn't affect the typings at all.

Reply:
> This made me wonder if there's a recommendation for ECS to avoid including fields altogether if they are not defined.

I didn't add them not to clutter logs with useless information. The serialized log record for `trace: { id: traceId }` in the case of `traceId: undefined` would be `trace: {}`